dist/components/Toast/index.d.ts

```diff
@@ -6,6 +6,8 @@ declare type Props = {
     label: string;
     text?: string;
     type: 'info' | 'warning';
+    index: number;
+    onClickClose: (index: number) => void;
```

Comment:
Toast自体はindexを知らず`onClickClose: () => void;`だけでもいいのではないのかっと思ったのですが、いかがでしょうか

Reply:
@hrb-suzuki 確認ありがとうございます！Toastをリストとして扱うので、どのToastのCloseボタンが押されたのかを呼び出し側に知らせる必要があるかと思いました。
```suggestion
    onClickClose: (index: number) => void;
```
だけでもどのToastが押されたのかというのを使用者に知らせることってできますかね？？

Reply:
呼び出し側で
```tsx
<Toast
  onClickClose={() => onClickClose(index)}
/>
```
に準ずる事(arrow functionそのままはだめ)をするだけだと思います。

> どのToastのCloseボタンが押されたのかを呼び出し側に知らせる必要があるかと思いました。 

Toast視点でいうと自分を押されているだけなので、自分が押された事しかわからない気がします。

Reply:
リストとして扱うのは扱う側の勝手な都合でToastのComponent自体は把握していない方が良いかと思いました。

(これがToastではなくToastListItemみたいな名前のCopmponentなら別かもしれませんが)

Reply:
@hrb-suzuki 閉じる際の処理とそのindexを呼び出し側から渡すイメージでしょうか？

>Toast視点でいうと自分を押されているだけなので、自分が押された事しかわからない気がします。
ということは、Props`index:number`も持たないようにする形が理想ですかね。

Reply:
そうですね必要なくなるかと思います。

Reply:
@hrb-suzuki ありがとうございます。修正いたしました。